File: src/main/java/com/etour/app/invoice/dto/InvoicePassengerDTO.java
package com.etour.app.invoice.dto;

import java.math.BigDecimal;

public class InvoicePassengerDTO {

    private String passengerName;
    private String passengerType;
    private BigDecimal passengerAmount;

    public InvoicePassengerDTO() {}

    public String getPassengerName() {
        return passengerName;
    }

    public void setPassengerName(String passengerName) {
        this.passengerName = passengerName;
    }

    public String getPassengerType() {
        return passengerType;
    }

    public void setPassengerType(String passengerType) {
        this.passengerType = passengerType;
    }

    public BigDecimal getPassengerAmount() {
        return passengerAmount;
    }

    public void setPassengerAmount(BigDecimal passengerAmount) {
        this.passengerAmount = passengerAmount;
    }
}

File: dotnet/backend/DTO/Invoice/InvoicePassengerDTO.cs
namespace Etour_Backend_dotnet.DTO.Invoice;

public class InvoicePassengerDTO
{
    public int PassengerId { get; set; }
    public string PassengerName { get; set; } = null!;
    public DateOnly DateOfBirth { get; set; }
    public string Gender { get; set; } = null!;
    public string PassengerType { get; set; } = null!;
    public decimal Amount { get; set; }
}
